rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -83,8 +83,13 @@ def _init_db_with_target_version(
         'rotkehlchen.db.dbhandler.DB_SCRIPT_CREATE_TABLES',
         new='',
     )
+    no_sanity_check = patch(
```

Comment:
Since this is used in multple places perhaps abstract it in an utill function or a fixture?